content/bn/docs/reference/glossary/annotation.md

```diff
@@ -0,0 +1,16 @@
+---
+title: এনোটেশন
+id: annotation
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/annotations
+short_description: >
+  অবজেক্টগুলির উপর অনিয়মিতভাবে অপরিচিত মেটাডেটা সংযুক্ত করার জন্য ব্যবহৃত একটি কী-মান জোড়।
+aka: 
+tags:
+- fundamental
+---
+অবজেক্টগুলির উপর অনিয়মিতভাবে অপরিচিত মেটাডেটা সংযুক্ত করার জন্য ব্যবহৃত একটি কী-মান জোড়।
+
+<!--more--> 
+
+এনোটেশনের মেটাডেটা ছোট অথবা বড়, বিন্যাসিত বা অবিন্যাসিত হতে পারে, এবং এটিতে {{< glossary_tooltip text="labels" term_id="label" >}} দ্বারা অনুমোদিত নয় এমন ক্যারেক্টার থাকতে পারে। টুল এবং লাইব্রেরির মতো ক্লায়েন্ট এই মেটাডেটা পুনরায় নিতে পারে।
```

Comment:
এনোটেশনের মেটাডেটা ছোট অথবা বড়, বিন্যাসিত বা অবিন্যাসিত হতে পারে, এবং এটিতে {{< glossary_tooltip text="লেবেল" term_id="label" >}} দ্বারা অনুমোদিত নয় এমন ক্যারেক্টার থাকতে পারে। টুল এবং লাইব্রেরির মতো ক্লায়েন্ট এই মেটাডেটা পুনরুদ্ধার করতে পারে।

Reply:
@asem-hamid you can write suggestions that are committable, eg:
```suggestion
এনোটেশনের মেটাডেটা ছোট অথবা বড়, বিন্যাসিত বা অবিন্যাসিত হতে পারে, এবং এটিতে {{< glossary_tooltip text="লেবেল" term_id="label" >}} দ্বারা অনুমোদিত নয় এমন ক্যারেক্টার থাকতে পারে। টুল এবং লাইব্রেরির মতো ক্লায়েন্ট এই মেটাডেটা পুনরুদ্ধার করতে পারে।
```